### PR TITLE
Select first project

### PR DESCRIPTION
**Pull request by vguzmanp**

### CODE REVIEW

File: Tests/State/TestProjectsManagerState.cs [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+				futureProject,
+				nowProject,
+				pastProject,
+			};
+
+			await state.LoadState (null);
+
+			CollectionAssert.AreEqual (sortedProjectList, state.ViewModel.Model);
+		}
+
```
Comment (SandraPG85) [inline Tests/State/TestProjectsManagerState.cs]: the name of the tests are not following the nomenclature: MethodName_StateUnderTest_ExpectedBehavior

Comment (SandraPG85) [inline Tests/State/TestProjectsManagerState.cs]: Add also a test when the collection is empty
